README.markdown

```diff
@@ -1,42 +1,45 @@
-# Spring RTS game engine
-[![Build Status](https://travis-ci.org/spring/spring.svg?branch=develop)](https://travis-ci.org/spring/spring)
+# BeyondAllReason RTS game engine
 
-## README
+    git clone https://github.com/beyond-all-reason/spring
```

Comment:
`git clone https://github.com/beyond-all-reason/spring -b BAR105`

Reply:
But isn't that outdated in like no time?